untidy/contaminators.py

```diff
@@ -1,9 +1,109 @@
 """ Functions to contaminate the data """
+import pandas as pd
+import numpy as np
+import random
 
 """ Column-wise contaminators """
 
+def data_check(data):
+    """
+    Check input data
+
+    Params
+    ------
+    data: pd.Series, pd.DataFrame
+        data to check
+
+    Returns
+    -------
+    row_idxs, col_idxs: int(s)
+        integers specifying, respectively, the amount of rows and columns in data.
+        These values will be passed to *random_indeces* in order to generate the indeces to be contaminated
+
+    """
+    if isinstance(data, pd.Series):
+        row_idxs = data.shape[0]
+        col_idxs = 2
+
+    elif isinstance(data, pd.DataFrame):
+        row_idxs, col_idxs = data.shape 
+        
+    else:
+        raise TypeError('data should be pd.Series or pd.DataFrame')
+
+    return row_idxs, col_idxs
+
+def random_indices(row_idxs, col_idxs, corruption_level = 4):
+    """
+    Get random indeces to contaminate
+
+    Parameters
+    ----------
+    row_idxs: int
+        number of rows in data to contaminate
+    col_idxs: int
+        number of columns in data to contaminate
+    corruption_level int, optional
+        level of corruption, should be between 0 and 10, where 0 leaves the dataset as is, 10
+        is the highest level of contamination
+
+    Returns
+    -------
+    list of indeces to be contaminated
+    """
+
+    # define amount of missing values to introduce
+    missing_count = int(row_idxs * col_idxs * corruption_level/10)
```

Comment:
this means that in case of corruption level 10 all the cells would become NaNs? I think that's a bit too much. We should come up with a reasonable maximum contamination

Reply:
Yes, we need to discuss this. Likely 80% will be enough